sources/tools/Stride.ProjectGenerator/Stride.ProjectGenerator.csproj

```diff
@@ -8,12 +8,6 @@
     <StrideBuildTags>WindowsTools</StrideBuildTags>
     <StrideCompilerTargetsEnable Condition="'$(StridePackageBuild)' == 'true'">false</StrideCompilerTargetsEnable>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
-    <Prefer32Bit>true</Prefer32Bit>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-    <Prefer32Bit>true</Prefer32Bit>
-  </PropertyGroup>
```

Comment:
`C:\Program Files\dotnet\sdk\8.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.RuntimeIdentifierInference.targets(275,5): warning NETSDK1189: Prefer32Bit is not supported and has no effect for netcoreapp target.`